src/Functions/Public/relocation/New-vRAOnboardingDeployment.ps1

```diff
@@ -97,7 +97,7 @@
                 $URI = "/relocation/onboarding/deployment"
                 $OnboardingDeployment = Invoke-vRARestMethod -Method POST -URI $URI -Body $Body -Verbose:$VerbosePreference
 
-                CalculateOutput
+                CalculateOutput($OnboardingDeployment)
```

Comment:
Should the parameter be positional here and not in ()?

Reply:
Should the parameter be positional here and not in ()?